libnavigation-base/build.gradle

```diff
@@ -15,6 +15,7 @@ android {
 }
 
 dependencies {
+    compileOnly(project(':annotations'))
```

Comment:
`kapt`?

Reply:
This should be removed all together. Also, `kapt` is for the processor, not annotations :)